src/features/Main/components/OurClassText/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from 'react';
+
+const OurClassText: FC = () => {
+  return (
+    <div className="flex justify-center items-center my-10 mx-4 md:mx-2 ">
+      <div className="text-wrap w-[750px]">
+        <h1 className="font-semibold text-[#00CCFF] text-center text-[20px] md:text-[30px]">
```

Comment:
if you've got a custom color, feel free to add it to `tailwind.config.ts`, 
for example: 
```
theme: {
    extend: {
        colors: {
            "primary-blue": "#00CCFF",
         },
    }, 
}
```

Reply:
@tjoetmuty check the color on the tailwind config, if it already exists, use the existing color

Reply:
@vickyadrii finished 3df8a46f2b9a8c523816326fde1d9f5b305aca6e

Reply:
@mpsalunggg the custom color for it didnt exist yet, so i added it. and for button "learn more", i havent finished but im still working on it